Formula/confluent-hub-client.rb

```diff
@@ -0,0 +1,13 @@
+class ConfluentHubClient < Formula
+  desc "Confluent Hub Client"
+  homepage "https://confluent.io"
+  url "https://s3-us-west-2.amazonaws.com/confluent-hub-client/confluent-hub-client-5.1.0-beta180619025141-package.tar.gz"
+  version "5.1.0-beta180619025141"
+  sha256 "6520a3bda6cbcf2b695e0d46248ea1638e1039b6f0a8e47a1a9ac3af5c21a388"
+
+  def install
+    bin.install "bin/confluent-hub"
```

Comment:
Don't we need a prefix? How do we get the location for CP? 
It's been a while since I wrote one for confluent-cli, I might be missing something

Reply:
This formula manifests installation of Confluent Hub client into Brew's Cellar - `/usr/local/Cellar/confluent-hub-client`. WRT to a location of CP, the client has `--worker-configs` and `--component-dir` options to specify the desired configuration. Also, the client has detection logic, which detects configs locations and default locations.